build/includes/release.mk

```
@@ -63,7 +63,7 @@ release-deploy-site: DOCKER_RUN_ARGS += -e GOFLAGS="-mod=mod" --entrypoint=/usr/
 release-deploy-site:
 	version=$$($(DOCKER_RUN) run $(mount_path)/build/scripts/previousversion/main.go -version $(base_version)) && \
 	echo "Deploying Site Version: $$version" && \
-	$(MAKE) site-deploy SERVICE=$$version
+	$(MAKE) site-deploy SERVICE=$$version HUGO_ENV=production
```

```suggestion
	$(MAKE) site-deploy SERVICE=$$version HUGO_ENV=snapshot
```

Since this is what we call at release time to capture the site snapshot